doc/misc.rst

```diff
@@ -133,17 +133,24 @@ and it is packaged so that the OpenCL drivers that are registered in the OS
 using registry keys are automatically available.
 
 
-Installing from PyPI with Linux wheels
---------------------------------------
+Installing from PyPI wheels
+---------------------------
+
+PyOpenCL distributes wheels for Python 3.6-3.10 for macOS, Windows, GLIBC base Linux and musl base Linux.
+For GLIBC base linux minimum version of GLIBC is 2.17. 
+There are also distributted wheels for PyPy 3.7-3.9 but only for GLIBC based linux 
```

Comment:
I agree that this information will likely not age well, especially the reference to specific platforms and version numbers. Maybe just link to the pypi page? People can see themselves whether a wheel is available for their platform. I think less specific is the way to go here.